src/libostree/ostree-bootloader-uboot.c

```diff
@@ -106,6 +106,7 @@ create_config_from_boot_loader_entries (OstreeBootloaderUboot *self, int bootver
   g_autoptr (GPtrArray) boot_loader_configs = NULL;
   OstreeBootconfigParser *config;
   const char *val;
+  g_autofree char *bootdir = NULL;
```

Comment:
It might be better if we moved the scope of this variable into the for loop, like the "index_suffix" variable.